src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrBackflower.cs

```diff
@@ -195,7 +209,7 @@ internal class VmrBackFlower : VmrCodeflower, IVmrBackFlower
             foreach (VmrIngestionPatch patch in patches)
             {
                 // TODO: Catch exceptions?
-                await _vmrPatchHandler.ApplyPatch(patch, repoPath, discardPatches, cancellationToken);
+                await _vmrPatchHandler.ApplyPatch(patch, targetRepo.Path, discardPatches, cancellationToken);
```

Comment:
A nit: shouldn't the ApplyPatch also take the repo, rather than a path? Both are valid, but I believe the intent is top apply patch to a repo, not just any path. A nit though as it's just about expressing the intent, not the functionality.

Reply:
We are actually applying to paths within repos and it is very important - e.g. inlining submodules, splitting >1GB patches etc. All of these need to target a folder.

Reply:
Makes sense, thanks for clarification.